lib/recurly/billing_info.php

```diff
@@ -23,6 +23,8 @@
  * @property string $verification_value Security code or CVV, 3-4 digits STRONGLY RECOMMENDED
  * @property string $ip_address Customer's IP address when updating their Billing Information STRONGLY RECOMMENDED
  * @property string $external_hpp_type Used to indicate payment made out of band via an external service (e.g. Adyen HPP).
+ * @property string $gateway_token The token generated by the payment gateway (Vantiv for example). When using this attribute you must set gateway_code. Also, billing_info.month will be the month of the token's expiration instead of the card's expiration month, and billing_info.year will be the year of the token's expiration instead of the card's expiration year.
+ * @property string $gateway_code The base36 encoded id of the gateway to use. See gateway_token.
```

Comment:
I don't think we want to describe the origin on the gateway code, better to just leave it as an opaque gateway identifier. We can probably say something like they can find it listed on the Payment Gateway in the UI.

Reply:
Good idea